include/class.staff.php

```diff
@@ -88,6 +88,7 @@ function getConfig() {
                         'default_from_name' => '',
                         'datetime_format'   => '',
                         'thread_view_order' => '',
+                        'img_att_view' => 'Download',
```

Comment:
I think it should default to `New-Tab` - which by the way is a weird choice of word. It should be `inline` in my humble opinion.

Reply:
I disagree with the defaulting to New Tab...we implemented Download by default so that by default Agent's aren't susceptible to malicious/malformed images. If Agents want to be susceptible to possibly tainted images then they can change it to New Tab (or `Inline` as it's called now).

I have updated the wording to `Inline` instead. I agree that this is more understandable and much better wording.